### DISCUSSION TITLE
How to get chapter wise html pages from publication server instead of getting all the html pages of all chapter

### DISCUSSION THREAD

**Discussion by shrey-creator** [readium/architecture#186] [Q&A]
Hey everyone,
I have cloned dita reader which is implementation of Readium reader. Now when i open the epub book it takes about 10-20 seconds to fetch all the chapters in book and then save it in cache. So is there a way to make those API calls either parallel or fetch one chapter at a time to reduce initial load time

**Comment by danielweck** [comment]
Please file an issue in the DITA reader GitHub repository: https://github.com/d-i-t-a/R2D2BC

Thank you :)
